org.eclipse.xtext.tests/src/org/eclipse/xtext/xtext/GrammarFlatteningTest.xtend

```diff
@@ -3,7 +3,7 @@
  * This program and the accompanying materials are made available under the
```

Comment:
This file is very hard to review because of the changed indentation of the test expectation. Can you please restore the original indentation?